From aa1c7a8fd783ed64cc8a35fba925c23672fe4885 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Fri, 25 Aug 2017 12:45:16 +0530
Subject: [PATCH 118/155] board: common:vid: Add LS1088A VID Supported voltage
 values

Adds below voltage values supported by LS1088A Soc:

1.025 V(default), 0.9875V, 0.9750 V, 0.9V, 1.0 V, 1.0125 V, 1.0250 V

Signed-off-by: Ashish Kumar <Ashish.Kumar@nxp.com>
Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 board/freescale/common/vid.c | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/board/freescale/common/vid.c b/board/freescale/common/vid.c
index 9b65c13b1a..bbcf475511 100644
--- a/board/freescale/common/vid.c
+++ b/board/freescale/common/vid.c
@@ -296,15 +296,28 @@ int adjust_vdd(ulong vdd_override)
 	unsigned long vdd_string_override;
 	char *vdd_string;
 	static const uint16_t vdd[32] = {
+#ifdef CONFIG_ARCH_LS1088A
+		10250,
+		9875,
+#else
 		10500,
 		0,      /* reserved */
+#endif
 		9750,
 		0,      /* reserved */
+#ifdef CONFIG_ARCH_LS1088A
+		0,      /* reserved */
+#else
 		9500,
+#endif
 		0,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
+#ifdef CONFIG_ARCH_LS1088A
+		9000,
+#else
 		0,      /* reserved */
+#endif
 		0,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
@@ -313,10 +326,18 @@ int adjust_vdd(ulong vdd_override)
 		0,      /* reserved */
 		0,      /* reserved */
 		10000,  /* 1.0000V */
+#ifdef CONFIG_ARCH_LS1088A
+		10125,
+#else
 		0,      /* reserved */
+#endif
 		10250,
 		0,      /* reserved */
+#ifdef CONFIG_ARCH_LS1088A
+		0,      /* reserved */
+#else
 		10500,
+#endif
 		0,      /* reserved */
 		0,      /* reserved */
 		0,      /* reserved */
-- 
2.14.1

